src/main/java/seedu/address/model/server/Server.java

```diff
@@ -0,0 +1,47 @@
+package seedu.address.model.server;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.commons.util.AppUtil.checkArgument;
+
+public class Server {
+
+    private static final String VALIDATION_REGEX = "^(([0-9]|[1-9][0-9]|1[0-9][0-9]|2[0-4][0-9]|25[0-5])(\\.(?!$)|$)){4}$";
```

Comment:
Server can have more than just numbers

Reply:
for example we can use this format: ms/name@1.1.1.1:25565 or ms/@1.1.1.1:25565 where name is optional, server domain name/ip address is compulsory, and port number is compulsory for ip address but not domain name: eg. ms/Mineplex@eu.mineplex.com

Reply:
server can have name and address, name is optional and address is compulsory.